crt_portal/cts_forms/views.py

```diff
@@ -18,11 +18,18 @@ def IndexView(request):
     # formatting protected class
     for report in latest_reports:
         reports = []
-        for p_class in report.protected_class.all().order_by('-form_order'):
+        for p_class in report.protected_class.all().order_by('form_order'):
             if p_class.protected_class is not None:
```

Comment:
Having to call `p_class.protected_class` makes the code difficult to read; it's hard to know how or why the second `protected_class` is different than the first `p_class`.

Looking at the `ProtectedClass` model, I'm seeing that the `protected_class` field is a character field. I'm thinking this is really the name of the protected class, not the protected class of the protected class — does that sound right to you, too?

What do you think about a refactor to change the field to `name`? Since we're in the early stages of building out the app, a refactor for clearer field names could improve the code in a big way going forward.

Reply:
We talked this through and decided that renaming the field to `protected_class_name` could be a good move for a follow-on PR.